core/src/main/scala/io/chrisdavenport/epimetheus/NameCommons.scala

```diff
@@ -5,8 +5,9 @@ import cats.implicits._
 
 // Mixed into Scala version-specific companion objects
 trait NameCommons {
-  implicit val nameInstances: Show[Name] with Semigroup[Name] with Eq[Name] with Order[Name] =
-    new Show[Name] with Semigroup[Name] with Eq[Name] with Order[Name]{
+  implicit val nameInstances
```

Comment:
would it be possible to remove this whitespace change?

Reply:
fixed